init.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Find the absolute script location dir (e.g. BASEDIR=/opt/openhabian)
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do
+  BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$BASEDIR/$SOURCE"
+done
+BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+SCRIPTNAME="$(basename "$SOURCE")"
+
+CONFIGFILE="/etc/openhabian.conf"
+#REPOSITORYURL=$(grep -i '^repositoryurl' ${CONFIGFILE} | cut -d '=' -f2)
+#CLONEBRANCH=$(grep -i '^clonebranch' ${CONFIGFILE} | cut -d '=' -f2)
+#JAVA_ARCH=$(grep -i '^java_arch' ${CONFIGFILE} | cut -d '=' -f2)
+#MODE=$(grep -i '^mode' ${CONFIGFILE} | cut -d '=' -f2)
+
+export SOURCE BASEDIR SCRIPTNAME CONFIGFILE # REPOSITORYURL CLONEBRANCH JAVA_ARCH MODE
```

Comment:
as stated already, it might be hard to locate init.bash for sourcing - actually this is the first part of this file what you actually need to execute in the script which wants to source init.bash.... 
given that trouble, consider removing it.... see comment above.

Reply:
ok, removed it